src/pages/docs/administration/managing-infrastructure/moving-your-octopus/move-the-database-and-server.md

```diff
@@ -34,6 +38,11 @@ Below are instructions on how to move your Octopus Server and SQL Database.
    - Task Logs
    - Packages
       - This folder only needs to be moved if using the built-in package repository. External feed details are stored in the database, and they will connect automatically.
+   - Event Exports
+
+:::div{.hint}
```

Comment:
I don't think this hint block is needed here

Reply:
Deleted 😊